### ISSUE TITLE
[SampleCard-Restaurant; Config-windows-timeline]: The color contrast ratio for the text is 2.8:1 which does not meet requirement of at least 4.5:1

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7398]
https://microsoft.visualstudio.com/OS/_workitems/edit/38020541
